### DISCUSSION TITLE
Android app search fails sporadically

### DISCUSSION THREAD

**Discussion by novedevo** [community/community#38363] [Mobile]
Google pixel 2xl, android 11, searching for "gopher" caused it this time.

![Screenshot_20221105-093907.png](https://user-images.githubusercontent.com/20878868/200131384-abcb8a30-d322-4499-a47d-3a056f882bbc.png)

**Comment by david-engelmann** [comment]
Do you have the most up to date version of the app?

**Reply by novedevo** [reply]
Yes

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @novedevo ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by metecanpolat1** [comment]
Tamam

**Comment by aconsuegra** [comment]
Hi @novedevo, thanks for the report. This is caused by a bug on our side, we will try to work on a fix soon!

**Reply by metecanpolat1** [reply]
****
